R/build-manual.R

```diff
@@ -12,7 +12,7 @@ build_manual <- function(pkg = ".", path = NULL) {
   name <- paste0(pkg$package, "_", pkg$version, ".pdf", collapse = " ")
```

Comment:
Another place for `glue()`? I'm not too sure what the point of `collapse = " "` is here 🤔 it doesn't seem to be doing anything.

Reply:
True enough!